concent_api/core/message_handlers.py

```diff
@@ -1142,8 +1142,8 @@ def handle_messages_from_database(client_public_key: bytes) -> Union[message.Mes
                 payment_ts=parse_datetime_to_timestamp(payment_message.payment_ts),
                 task_owner_key=payment_message.task_owner_key.tobytes(),
                 provider_eth_account=payment_message.provider_eth_account,
-                amount_paid=payment_message.amount_paid,
-                amount_pending=payment_message.amount_pending,
+                amount_paid=int(payment_message.amount_paid),
```

Comment:
Same here, add properties to `PaymentInfo` model.

Reply:
Done